modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -124,6 +123,16 @@ const NotificationsInfo = ({
 		setRecipientType(item.recipientType);
 	};
 
+	if (selectedItem?.type === 'task' && recipientTypeOptions.length < 6) {
```

Comment:
@MariMalta It would be better to check directly if the taskAssignees option is already on the array, instead of relying on the array length. You can replace this if and the next one with the following code:
```
if (
		selectedItem?.type === 'task' &&
		!recipientTypeOptions
			.map((option) => option.value)
			.includes('taskAssignees')
	) {
		recipientTypeOptions.push({
			label: Liferay.Language.get('task-assignees'),
			value: 'taskAssignees',
		});
	} else if (selectedItem?.type !== 'task') {
		recipientTypeOptions = recipientTypeOptions.filter(
			(option) => option.value !== 'taskAssignees'
		);
	}
```